src/Sulu/Bundle/AdminBundle/Resources/js/services/DOM/afterExecutionQueue.js

```diff
@@ -0,0 +1,4 @@
+// @flow
+export default function afterExecutionQueue(callback: () => *) {
```

Comment:
You are right. I called it `afterElementsRendered` as the in my opinion "after" better captures the notion of a callback being called after something has happened.